includes/calendars/views/default-calendar-grid.php

```diff
@@ -164,6 +166,20 @@ public function styles()
 		];
 	}
 
+	/**
+	 * Format the timestamp with timeZone to display exact month as per the diffrent timezone.
+	 *
+	 * @since  3.4.3
+	 */
+	public function format_timestamp($format = 'F', $timestamp)
```

Comment:
I remember that we have this function already somewhere, why are defining again?